Implement sortKey direction argument

srdante opened the pull request:
This pull request implements #21 with the following syntax:

```php
$table->integer('price')->sortKeyDesc();

$table->sortKeyDesc('column');

$table->sortKeyDesc(['column1', 'column2']);
```

carlsverre commented on tests/Hybrid/CreateTable/SortKeysDescTest.php:
```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * @author Aaron Francis <aarondfrancis@gmail.com|https://twitter.com/aarondfrancis>
```
You can update this to your info or remove as you like.

carlsverre commented on src/Schema/Blueprint/ModifiesIndexes.php:
```diff
@@ -29,6 +29,16 @@ public function sortKey($columns, $name = null)
+     * @param $name
+     * @return \Illuminate\Support\Fluent
+     */
+    public function sortKeyDesc($columns, $name = null)
```
since sortKey is a SingleStore specific thing (I am pretty sure Laravel doesn't support this normally) - what do you think about adding the direction as an optional parameter to the main `sortKey` method? Similar to the name argument?

Something like this:
```php
    public function sortKey($columns, $name = null, $direction = "asc")
```

carlsverre commented:
Also, thank you so much for sending in this PR! I really appreciate it. I am planning on spending some time with the Laravel connector next week to finalize this and other open issues. Will make sure to get this in asap.

srdante commented:
@carlsverre Updated! I'm not really a fan of adding it as an argument since the second argument is the index name. If I want to set the index and the direction without changing the name, I will always need to use named arguments or skip the second one:

```php
$table->sortKey(columns: 'name', direction: 'desc');

// OR

$table->sortKey('name', null, 'desc');
```

srdante commented:
@carlsverre After I saw the code again, I noticed the fluent API was not working correctly, and the index name parameter was not being used. So I removed the index name parameter for the shard & sort key and fixed the fluent API to support the 'desc' parameter:

```php
$table->sortKey('price', 'desc');

$table->integer('price')->sortKey('desc');
```
